lib/Doctrine/DBAL/Connection.php

```diff
@@ -467,6 +467,11 @@ private function getServerVersion()
             return $this->_conn->getServerVersion();
         }
 
+        // if \PDO
+        if($this->_conn instanceof \PDO){
```

Comment:
```suggestion
        if ($this->_conn instanceof \PDO){
```

Reply:
I have tried to re-edit and submit, please review.